Source/PGDebugViewController.swift

```diff
@@ -142,7 +162,7 @@ public class PGDebugViewController: UIViewController {
     
     func selectModule(at index: Int) {
         func present(_ title: String, modules: [PGDebuggableData]) {
-            let debugVC = PGDebugViewController(cellModules: modules)
+            let debugVC = PGDebugViewController(cellModules: modules,path: self.plistPath ?? "")
```

Comment:
This is not a part of production code but, please think this is a practice. 
Can you add white space between comma, and modules? 
`(cellModules: modules, path: self.plistPath ?? "")`